.env.base

```diff
@@ -41,7 +41,7 @@ REACT_APP_FEATURE_LIFI_SWAP=true
 REACT_APP_FEATURE_ONE_INCH=true
 REACT_APP_FEATURE_THOR_SWAP=true
 REACT_APP_FEATURE_THOR_SWAP_STREAMING_SWAPS=true
-REACT_APP_FEATURE_THOR_SWAP_AFFILIATE_FEES=true
+REACT_APP_FEATURE_THOR_SWAP_AFFILIATE_FEES=false
```

Comment:
Just a quick note here to remind reviewers, the logic inside thorchain swapper has changed slightly so disabling this will need a retest of the swapper with and without donations checked